atk v2.56.4

conda-forge-admin opened the pull request:
Hi! This is the friendly automated conda-forge-webservice.

I've started rerendering the recipe as instructed in #23.

If I find any needed changes to the recipe, I'll push them to this PR shortly. Thank you for waiting!



Here's a checklist to do before merging.
- [ ] Bump the build number [if needed](https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-recipes)

Fixes #23

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17585307308](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17585307308). Examine the logs at this URL for more detail.</sub>

h-vetinari committed:
minor recipe clean-ups

h-vetinari committed:
add dev_url

h-vetinari committed:
atk v2.57.2

h-vetinari committed:
add zlib

h-vetinari committed:
cross-compile aarch/ppc

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.52.1 and conda-forge-pinning 2025.09.09.11.23.21

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

tschoonj commented:
@h-vetinari  this is a development release, and will not be merged in

h-vetinari commented:
> @h-vetinari this is a development release, and will not be merged in

Right. Even releases should be good though, right? https://gitlab.gnome.org/GNOME/at-spi2-core/-/releases/2.56.4

h-vetinari committed:
atk v2.56.4

h-vetinari committed:
add cmake

tschoonj commented:
yep, that sounds good.

h-vetinari committed:
add dbus

h-vetinari committed:
add libxml2

h-vetinari committed:
MNT: Re-rendered with conda-smithy 3.52.1 and conda-forge-pinning 2025.09.09.12.58.34

Other tools:
- conda-build 25.7.0
- rattler-build 0.47.0
- rattler-build-conda-compat 1.4.6

h-vetinari committed:
no dbus on windows

h-vetinari committed:
sync updates from gdk-pixbuf-feedstock

see https://github.com/conda-forge/gdk-pixbuf-feedstock/blame/main/recipe/build.sh

h-vetinari committed:
remove flag that's superfluous with `-Dbuildtype=release` in MESON_ARGS

h-vetinari committed:
remove some unrelated options
